plugins/database/src/db_plugin.cpp

```diff
@@ -15810,6 +15810,67 @@ auto db_check_auth_credentials_op(irods::plugin_context& _ctx,
     }
 } // db_check_auth_credentials_op
 
+auto db_execute_genquery2_sql(irods::plugin_context& _ctx,
+                              const char* _sql,
+                              const std::vector<std::string>* _values,
+                              char** _output) -> irods::error
```

Comment:
This allows admins to see the SQL within policy, but not the values. The reason being because `_values` is not a serialized type.

Reply:
Perhaps this can be resolved by adding logic to re_serialization.cpp. The only issue is all serialized args tend to be map-like objects. We need to represent this as a list.

Reply:
Agreed. Now the question becomes, how should the vector be serialized?

Are we building new MSIs? _(This takes time. Also feels too specific to vector<string>.)_

If not, our only option is re_serialization.cpp. That means the vector is transformed into a map-like object. Imagine the following:
```
*arg0 = *VALUES."0";
*arg1 = *VALUES."1";
```
Looking at the re_serialization.cpp case, it doesn't look too bad. It's also simple to implement.

Thoughts?

Reply:
I've added serialization for `[const] vector<string>*`. These are types will appear as map-like objects as described. The expose each element of the vector and the number of elements.

See the tests in `test_dynamic_peps.py` for details.